tests-functional/clients/status_backend.py

```diff
@@ -61,9 +61,12 @@ def rpc_valid_request(self, method, params=[], _id=None, url=None):
         self.verify_is_valid_json_rpc_response(response, _id)
         return response
 
+    def verify_response_schema(self, response, method):
+        self.verify_json_schema(response.json(), method)
+            
     def verify_json_schema(self, response, method):
         with open(f"{option.base_dir}/schemas/{method}", "r") as schema:
```

Comment:
I don't see any particular reason for having 2 methods that are almost the same. Let's leave only one

Reply:
Yes, I see these methods are being used in different tests. But I still suggest using only one and redoing other tests accordingly